lib/builtin.pm

```diff
@@ -371,6 +372,25 @@ This must be called at compile time; which typically means during a C<BEGIN>
 block. Usually this would be used as part of an C<import> method of a module,
 when invoked as part of a C<use ...> statement.
 
+=head2 load_module
+
+    load_module($module_name);
+
+This function is currently B<experimental>.
+
+Loads a named module from the inclusion paths (C<@INC>).  C<$module_name> must
+be a string that provides a module name.  It cannot be omitted, and providing
+an invalid module name will result in an exception.  Not providing any argument
+results in a compilation error.  Returns a true value on success.
```

Comment:
Since the implementation and tests now work to the idea of returning the module name, these docs need fixing.

Reply:
Thanks, updated with:

> Returns the loaded module's name on success.